projects/1.19/assets/quark/quark/lang/zh_cn.json

```
@@ -1,1296 +1,1554 @@
+	"advancements.quark.crawl_in_hollow_log.description": "在空心原木中爬行",
+	"advancements.quark.get_forgotten_hat.title": "博物馆的老古董",
+	"advancements.quark.get_forgotten_hat.description": "在地下击杀一名遗忘者并缴获其帽子",
+	"advancements.quark.get_diamond_heart.title": "Shape of My Heart",
+	"advancements.quark.get_diamond_heart.description": "在地下潜行靠近石精灵并击碎它的外壳以获得钻石之心",
+	"advancements.quark.make_stoneling.title": "年轻的逐梦人",
+	"advancements.quark.make_stoneling.description": "通过对任意石头使用石精灵的钻石之心来给它一具崭新的躯体",
+	"advancements.quark.mine_toretoise.title": "矿石龟！",
+	"advancements.quark.mine_toretoise.description": "用镐子挖取野生矿石陆龟背上的矿物",
+	"advancements.quark.mine_fed_toretoise.title": "我们发财了！",
+	"advancements.quark.mine_fed_toretoise.description": "给矿石陆龟喂食发光浆果使其矿物重新生成，之后再次收获其背上的矿物",
+	"advancements.quark.throw_pickarang.title": "不知怎么的就回来了",
+	"advancements.quark.throw_pickarang.description": "掷出一枚皮卡镖",
+	"advancements.quark.retreive_flamerang.title": "Fly to the Sky, Fire Bird",
+	"advancements.quark.retreive_flamerang.description": "掷出一枚界炎镖，在其燃烧时取回一个物品",
+	"advancements.quark.overlevel_enchant.title": "超越极限",
+	"advancements.quark.overlevel_enchant.description": "使用古卷来超越魔咒的最高等级",
+	"advancements.quark.instamine_deepslate.title": "超越瞬间的极限",
+	"advancements.quark.instamine_deepslate.description": "同时使用效率VI和急迫II来瞬间破坏深板岩",
+	"advancements.quark.redirect_beacon.title": "See The Lights",
+	"advancements.quark.redirect_beacon.description": "使用刚玉簇改变信标光柱的路径",
+	"advancements.quark.get_bottled_cloud.title": "云存储",
+	"advancements.quark.get_bottled_cloud.description": "把云朵捕捉到瓶子中",
+	"advancements.quark.apply_color_rune.title": "雄伟者之矜持",
+	"advancements.quark.apply_color_rune.description": "使用铁砧将彩色符文应用到物品上",
+	"advancements.quark.wear_full_rainbow.title": "吃定彩虹",
+	"advancements.quark.wear_full_rainbow.description": "装备一身带有彩虹符文效果的盔甲",
+	"advancements.quark.throw_parrot_egg.title": "只是鹦鹉",
+	"advancements.quark.throw_parrot_egg.description": "给鹦鹉喂食甜菜种子，并投掷它生下的蛋",
+	"advancements.quark.pathfinder_map_center.title": "明日への道～Going my way～",
+	"advancements.quark.pathfinder_map_center.description": "到达寻路地图中X标号的位置",
+	"advancements.quark.get_own_head.title": "出人头地",
+	"advancements.quark.get_own_head.description": "利用女巫杀死已命名的雪傀儡，以此获取你自己的头",
+	"advancements.quark.ancient_apple_overlevel.title": "有点水果味",
+	"advancements.quark.ancient_apple_overlevel.description": "在自身经验等级超过100级的时候食用远古水果",
+	"advancements.quark.get_all_blossom_saplings.title": "花开正茂",
+	"advancements.quark.get_all_blossom_saplings.description": "获取所有种类的花树苗",
+	"advancements.quark.get_all_corundum.title": "刚玉难题",
+	"advancements.quark.get_all_corundum.description": "获取所有颜色的刚玉簇",
+	"advancements.quark.use_viaduct.title": "来追我啊",
+	"advancements.quark.use_viaduct.description": "将末影珍珠投掷到一系列幻境石水晶的一端，以此传送到另一端",
+	"advancements.quark.tame_foxhound.title": "玩火",
+	"advancements.quark.tame_foxhound.description": "在防火效果下喂食煤炭来驯服一只猎狐犬",
+	"advancements.quark.foxhound_speed_up_furnace.title": "让猎狐犬躺下",
```

“Let Flaming Dogs Lie”显然化用了“let sleeping dogs lie”，这句话有“警告别人少惹是非”的意思，我就翻译成了更有警告意味的这个
```suggestion
	"advancements.quark.foxhound_speed_up_furnace.title": "路边的烈犬你不要踩",
```

---

我觉得自家的猎狐犬没什么可警告的，可选用更自制一点的 `不要打扰猎狐犬`

---

这个也不错。